cmd/harbor/root/cveallowlist/cmd.go

```diff
@@ -0,0 +1,20 @@
+package cveallowlist
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func CVEAllowlist() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "cve-allowlist",
+		Short:   "Manage system CVE allowlist",
```

Comment:
Short:   "Manage system CVE allowlist",
		Long:    `Manage system level allowlist of CVE`,
		
The description is almost the same, maybe we can have a longer explanation.